_delphi_utils_python/tests/test_covidcast_wrapper.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime, timedelta
+from pathlib import Path
+import pandas as pd
+from delphi_utils import covidcast_wrapper
+import covidcast
+from freezegun import freeze_time
+from pandas.testing import assert_frame_equal
+
+TEST_DIR = Path(__file__).parent
+class TestCovidcastWrapper:
+    def test_metadata(self):
+        expected_df = covidcast.metadata()
+        df = covidcast_wrapper.metadata()
+        assert_frame_equal(expected_df, df)
+
+    @freeze_time("2024-07-29")
+    def test_signal(self):
+        meta_df = covidcast_wrapper.metadata()
```

Comment:
This is making an actual request over the network, but unit tests should be self-contained.  You could move these tests to `integrations/` but heavy use of external resources is discouraged there too...  The `delphi-epidata` client has a [debug mode](https://github.com/cmu-delphi/delphi-epidata/blob/8746ff2ef7a936bb93628bc1358471d7c6c4f5f8/src/client/delphi_epidata.py#L46) where it can tell you exactly what requests its making -- can you list all of the requests made by this test?  That can help give me an idea of how big of a lift this is doing.